about.html

```diff
@@ -0,0 +1,159 @@
+          </li>
+        </ul>
+      </div>
+      <div class="main">
+        <div class="invitation">
+          <div>
+            <img class="title" src="images/title.png" alt="title" />
+            <p class="message">
+              The Global Summit brings together the community of experts,
+              academics, and activists who comprise the Creative Commons
+              affiliate network in a different country every two. This year, we
+              hope to expand our invitation list including organizations and
+              individuals.
+            </p>
+            <p class="contact-us">
+              Please contact us per Email for any further questions <br />
+              about CC Global Summit 2015!
+            </p>
+            <a class="address" href="ccsummit2015@cckorea.org"
+              >ccsummit2015@cckorea.org</a
+            >
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <section class="creative-commons-wrap">
+      <div class="creative-commons">
+        <h2 class="title-creative">
+          Creative Commons <br />
+          Global Summit 2015 Logo
+        </h2>
+        <div class="guide_bar"></div>
+        <p class="creative-text">
+          The logo of CC Global Summit 2015 was dec ided through the logo
+          competition from 8, june to 7, july
+        </p>
+        <img
+          class="about-logo"
+          src="images/about_logo.png"
+          alt="ccsummit2015"
+        />
+      </div>
+    </section>
+
+    <section class="past-summit-wrap">
+      <div class="past-summit">
+        <h1 class="title-past">See the past CC Global Summits</h1>
+        <div class="guide_bar"></div>
+        <p class="past-text">
+          Take a look at the last two CC Global Summits which took place in
+          Buenos Aires and Warsaw.
+        </p>
+        <div class="picture-wrap">
+          <a href="#"
+            ><img
+              class="picture"
+              src="images/about_2013_hover.png"
+              alt="2013summit"
+          /></a>
+          <a href="#"
+            ><img
+              class="picture"
+              src="images/about_2011_hover.png"
+              alt="2011summit"
+          /></a>
+        </div>
+      </div>
+    </section>
+    <!-- Featured Speakers Section Ends -->
+    <section>
+      <div class="partners-wrap">
+        <div class="partners">
+          <h1>Partner</h1>
+          <div class="guide_bar"></div>
+          <img
+            class="partner_logos"
+            src="images/partner_logo.png"
+            alt="partner-logo"
+          />
+        </div>
+      </div>
+      <div class="copy-right-about">
+        <div class="cc">
+          <i class="fa fa-copyright fa-lg"></i>
+          <h4>creative <br />commons korea</h4>
+        </div>
+        <div class="year">
+          2015 Creative Commons Korea Some Rights Reserved.
+        </div>
+      </div>
```

Comment:
Some changes would be nice for you to implement here based on design specifications. They are:

- [ ] Kindly change the HTML tag `<div class="copy-right-about">` to a semantic `<footer class="copy-right-about>"` tag. 
- [ ] The `background-color` for the footer is `var(--black)` for the desktop version and `#fff` for the mobile version not `var(--gray)`.  For the mobile version, kindly use CSS declaration `gap` to give space between the logo and copyright text.

### Your Design (Desktop)

![image](https://user-images.githubusercontent.com/77106962/159128835-6d04fa42-43bd-41e2-87b2-daa7c85bc3f7.png)

### Your Design (Mobile)

![image](https://user-images.githubusercontent.com/77106962/159128915-e0b4bcd5-e004-4a69-b4a4-0d148f74ea3f.png)

### Behance Design (Desktop)

![image](https://user-images.githubusercontent.com/77106962/159128958-248237c5-e1e6-4825-8577-eb9ec937fc34.png)

### Behance Design (Mobile)

![image](https://user-images.githubusercontent.com/77106962/159129001-75946287-476d-43a8-a55f-5cff6ee00cfa.png)

Reply:
@Gambit142 Great insight and catches. Thank you for your detailed review and I will definitely correct them accordingly.